Makefile

```diff
@@ -29,13 +29,14 @@ watch: clean
 	BABEL_ENV=development ./node_modules/.bin/gulp watch
 
 lint:
-	./node_modules/.bin/eslint scripts packages *.js --format=codeframe --rulesdir="./eslint_rules"
+	./node_modules/.bin/eslint scripts packages *.js --format=codeframe
+	./node_modules/.bin/babel "./packages/babel-plugin-*/src" --plugins=transform-no-undefined-identifier --out-file=/dev/null
```

Comment:
`--plugins=transform-no-undefnied-identifier` doesn't work, because this plugins isn't installed inside `node_modules`.
Does `--plugins` support relative paths? If so, we can use `--plugins=./packages/babel-plugin-transform-no-undefined-identifier`

Reply:
seems like it does!

Reply:
hm, i guess then we need travis to build the plugin just to lint though